tools/local-setup/utils.sh

```
@@ -0,0 +1,53 @@
+        echo "Waiting for Graph Node to be ready..."
+        sleep 5
+    done
+    echo "Docker services are ready!"
+}
+
+deploy_contract() {
+    pnpm deployContract
+}
+
+update_subgraph() {
+    SCRIPT_PATH=$(realpath "$0")
```

Shellcheck (0.10.0)

[warning] SCRIPT_PATH appears unused. Verify use (or export if used externally).

(SC2034)